tests/test_utils.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
```

Comment:
dacht dat dit een python 2 restant was, mag weg indien dit niet nodig is? mss ineens overal

Reply:
Da's waar.